sdk/sdk-client/tests/queries/simulateRefinance.subtest.ts

```diff
@@ -92,23 +98,42 @@ export default async function simulateRefinanceTest() {
     }),
   }
 
-  const targetPool: SparkLendingPool = {
-    type: PoolType.Lending,
+  const targetPool = {
+    type: PoolType.Lending as const,
     protocol: protocol,
     poolId: {
       protocol: protocol,
     },
-    collaterals: {},
-    debts: {},
+    collaterals: CollateralConfigMap.createFrom({
+      record: {
+        [prevPosition.collateralAmount.token.address.value]: new CollateralConfigMock({}),
+      },
+    }),
+    debts: DebtConfigMap.createFrom({
+      record: {
+        [prevPosition.debtAmount.token.address.value]: new DebtConfigMock({}),
+      },
+    }),
     baseCurrency: DAI,
-  } as SparkLendingPool
+  }
 
+  const targetPosition = new Position({
```

Comment:
This is a bug that I introduced by mistake: please stick to using `Position.createFrom`, constructors should all be private (I removed the private access modifier when I thought it was affecting serialization)